community/import-tool/src/test/java/org/neo4j/tooling/ImportToolTest.java

```diff
@@ -920,12 +945,22 @@ private void writeNodeData( PrintStream writer, Configuration config, List<Strin
         {
             if ( linePredicate.accept( i ) )
             {
-                writer.println( nodeIds.get( i ) + delimiter + randomName() +
-                                delimiter + randomLabels( arrayDelimiter ) );
+                writer.println( getLine( nodeIds.get( i ), delimiter, arrayDelimiter, extraColumns ));
             }
         }
     }
 
+    private String getLine(String nodeId, char delimiter, char arrayDelimiter, int extraColumns) {
+        StringBuilder stringBuilder = new StringBuilder(  nodeId  + delimiter + randomName() +
+                delimiter + randomLabels( arrayDelimiter ));
```

Comment:
Don't do string concatenation along with a StringBuilder, this will compile down to using two StringBuilders, which is completely unnecessary.

Sure, this is "only a test", but we should hold those up to the same standard as the rest of the code. If for no other reason then at least so that we remember to always do it right.

Also: passing a string to the constructor of a StringBuilder typically isn't very useful since it'll set the buffer size of the StringBuilder to the size of the string, and you probably want the resulting string to be larger than that, so you want a larger buffer, otherwise you'll get more re-allocations of the buffer.
What is useful for passing into the constructor is a template string in which you will make substitutions.